includes/SpecialWitnessPublisher.php

```diff
@@ -13,21 +13,21 @@
 
 require_once 'Util.php';
 
-function shortenHash( $hash ) {
+function shortenWitnessHash( $hash ) {
```

Comment:
This is actually a general function that is used also in the data-accounting-external-verifier, hence the name is `shortenHash`, even though it is used only to shorten the witness hash here.

Reply:
I reverted the change